src/hooks/core/push-notification/usePushNotificationHook.ts

```diff
@@ -23,10 +24,11 @@ const usePushNotificationHook = () => {
   const navigation = useNavigation();
   const {signedProfileId} = useUserAuthHook();
   const {localDb} = useLocalDatabaseHook();
-  const {fetchChannelDetail, setSelectedChannel} = useChatUtilsHook();
+  const {selectedChannel, fetchChannelDetail, setSelectedChannel} = useChatUtilsHook();
   const {updateAppBadgeFromDB} = useAppBadgeHook();
 
   const [, setProfileAtom] = useRecoilState(profileAtom);
+  const [currentChatScreen] = useRecoilState(currentChatScreenAtom);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of Recoil state for `currentChatScreen` and the inclusion of `selectedChannel` in the destructuring assignment from `useChatUtilsHook()` are new additions. Make sure these states are managed properly throughout the component lifecycle.

```diff
+ const {selectedChannel, fetchChannelDetail, setSelectedChannel} = useChatUtilsHook();
+ const [currentChatScreen] = useRecoilState(currentChatScreenAtom);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->